taxi/views.py

```diff
@@ -35,6 +42,22 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
+    #
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        context = super(ManufacturerListView, self).get_context_data(**kwargs)
+        name = self.request.GET.get("name", "")
+        context["search_form"] = ManufacturerSearchForm(initial={"name": name})
+        return context
+
+    #
+
+    def get_queryset(self):
+        queryset = Manufacturer.objects.all()
```

Comment:
Consider adding a check to ensure that the `Car` object exists before attempting to filter the queryset. This can prevent potential errors if an invalid `pk` is provided.